Engine/SpecialVars.cs

```diff
@@ -114,6 +116,7 @@ static SpecialVars()
                                                                     /* ConfirmPreference */ typeof(ConfirmImpact),
                                                                     /* ProgressPreference */ typeof(Enum),
                                                                     /* InformationPreference */ typeof(ActionPreference),
+                                                                    /* ErrorView */         typeof(Enum), //ErrorView type not available on PS3
```

Comment:
There is ` PSV3` variable defined as we do conditional compilation for each version of PS (3, 4, 5 and 'Core'). You could use this to use Enum only for the v3 compiled version but the correct types for others.

Reply:
Okay I can do that, however I'm still not sure why this type array is here since nobody seems to reference it, should it be expanded despite this?

Reply:
@bergmeister @JamesWTruher what *is* the point of this enum? Can we just **delete** the whole thing, and merge this PR already?

Reply:
@Jaykul  Whilst it's not directly referenced, it is still needed as it clearly solves the issue of not flagging false positive for ErrorView. I presume the rule is using reflection. The `PSV3` variable is a pragma, i.e. a build time variable as we use different PowerShell version SDKs when we build against the different versions.

Reply:
Honestly, maybe we should probably just type this as string.

The `ErrorView` enum is new in PS7 and it's _not enforced_. The variable **can** be overwritten, and now that it's an Enum in PS7,  _it **has to be** overwritten_ by anyone who wants to improve the error view (such as my ErrorView module)...